benchmarks/bm_runner.py

```diff
@@ -537,6 +537,63 @@ def func(args: argparse.Namespace) -> None:
         _subprocess_runner([args.asv_sub_command, *args.asv_args], asv=True)
 
 
+class TrialRun(_SubParserGenerator):
+    name = "trialrun"
+    description = (
+        "Fast trial-run a given benchmark, to check it works : "
+        "in a provided or latest-lockfile environment, "
+        "with no repeats for accuracy of measurement."
+    )
+    epilog = (
+        "e.g. python bm_runner.py trialrun "
+        "MyBenchmarks.time_calc /tmp/testpython/bin/python"
+        "\n\nNOTE: 'runpath' also replaces $DATA_GEN_PYTHON during the run."
+    )
```

Comment:
I liked your idea of also including an example that uses $DATA_GEN_PYTHON - for people who aren't in the detailed headspace that we are, and might forget.

```
python bm_runner.py trialrun MyBenchmarks.time_calc $DATA_GEN_PYTHON
```